Automattic/vip-go-mu-plugins
# Update Jetpack 9.3 to 9.3.1

## Description

Update the `jetpack-9.3` subtree to version 9.3.1

## Changelog Description

### Plugin Updated: Jetpack 9.3.1

We upgraded Jetpack 9.3 to Jetpack 9.3.1.

Not a lot of significant changes in this patch release, just a couple of bugfixes:

https://github.com/Automattic/jetpack/releases/tag/9.3.1

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Go to the Jetpack dashboard in `wp-admin`
1. Ensure the available version is 9.3.1
